packages/application-generic/package.json

```diff
@@ -85,6 +85,7 @@
     "@novu/ses": "^0.20.0-alpha.0",
     "@novu/shared": "^0.20.0-alpha.0",
     "@novu/slack": "^0.20.0-alpha.0",
+    "@novu/guilded": "^0.16.3-alpha.0",
```

Comment:
I think this should be 0.20 like the rest. @davidsoderberg  can you correct me if I am wrong?

Reply:
I think 0.16.3 is my version that's why its reflecting in new provider config
Please tell me if change is required

Reply:
Yes it should be the same version as the rest. So please update it in your provider as well.

Reply:
ok will update and share